### PR TITLE
Fixed some validation messages and added new validation if worker is RM but not CQC

### PR DESCRIPTION
**Pull request by aaron-russell**

### CODE REVIEW

**Comment by aaron-russell** [comment]
[(3) BU Validation - Error - Non-CQC Regulated with Registered Manager](https://trello.com/c/emH7LdIl/26-3-bu-validation-error-non-cqc-regulated-with-registered-manager)

**Comment by aaron-russell** [comment]
[(1) BU Validation - Confirm error message text](https://trello.com/c/YLG74B0f/21-1-bu-validation-confirm-error-message-text)

File: server/models/BulkImport/csv/establishments.js [hunk 1]
```diff
@@ -1258,23 +1259,24 @@ class Establishment {
     const MAX_TOTAL = 999;
     const myTotalPermTemp = parseInt(this._currentLine.TOTALPERMTEMP, 10);
     const HEAD_OFFICE_MAIN_SERVICE = 72;
+    console.log('Total Perm Temp', this._currentLine.TOTALPERMTEMP);
```
Comment (mksrivastava) [inline server/models/BulkImport/csv/establishments.js]: I guess console is not required.

File: server/models/BulkImport/csv/establishments.js [hunk 2]
```diff
@@ -1258,23 +1259,24 @@ class Establishment {
...
 
-    if (Number.isNaN(myTotalPermTemp)) {
+    if (myTotalPermTemp.length === 0) {
       this._validationErrors.push({
         lineNumber: this._lineNumber,
         errCode: Establishment.TOTAL_PERM_TEMP_ERROR,
         errType: 'TOTAL_PERM_TEMP_ERROR',
-        error: 'Total Permanent and Temporary (TOTALPERMTEMP) must be an whole number',
+        error: 'TOTALPERMTEMP is missing',
```
Comment (mksrivastava) [inline server/models/BulkImport/csv/establishments.js]: I am not sure a single quote (') will work combinedly for variables and strings.

**Pull request merged**